src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -102,10 +108,112 @@ def validate_role(self, role):
         return role
 
 
+class OrganizationMemberIndexQueryParamSerializer(serializers.Serializer):
+    email = serializers.CharField(
+        required=False,
+        default=None,
+        help_text="an email address to filter by.",
+    )
+    scope = serializers.ChoiceField(
```

Comment:
unfortunately, our doc site doesn't actually show the choices of a choice field, but can see in swagger-ui
<img width="469" alt="Screen Shot 2022-02-24 at 1 38 53 PM" src="https://user-images.githubusercontent.com/1976777/155612078-5d5e5911-6402-4898-8a82-8623b837660d.png">

Reply:
edit: https://github.com/getsentry/sentry-docs/pull/4774